src/components/CaptionLegend.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
+import styled from '@emotion/styled';
 import PropTypes from 'prop-types';
 import ComponentFactory from './ComponentFactory';
-import { P2 } from './dev-hub/text';
+import { P3 } from './dev-hub/text';
+import { colorMap } from './dev-hub/theme';
 
-const Caption = P2.withComponent('figcaption');
+const StyledCaption = styled(P3)`
```

Comment:
are you using StyledCaption anywhere?

Reply:
yep! Using it on line 11 to also use the `figcaption` tag

Reply:
Are you trying to combine the styles of P3 with the color and then actually render a figcaption? I meant why can't like 8 just be `const Caption = styled('figcaption')`?

Reply:
Ah yes that is what I am looking to do, would it make more sense just to bring the P3 styles into a `styled('figcaption')` here?

Reply:
I guess this way is fine! just seemed weird to declare a style thats only used to declare another style but i think it does the trick!